ui/src/components/executions/Restart.vue

```diff
@@ -53,7 +53,7 @@
         <p v-html="$t(replayOrRestart + ' confirm', {id: execution.id})" />
 
         <el-form v-if="revisionsOptions && revisionsOptions.length > 1">
-            <p class="text-muted">
+            <p class="">
```

Comment:
Instead of removing the classes completely, I would suggest to add a class, let's say `.execution-description` and add `color: var(--bs-gray-700);` as a single property, like on this PR https://github.com/kestra-io/kestra/commit/9d1700caf112419acdcbc86d726193874831fe08#diff-6f86c9142fef96430e757017dc75d0d8674a20b61d1c45a6bbd7478332d84b84R151.

Reply:
Okay, Thanks for suggesting, will do that

Reply:
> Instead of removing the classes completely, I would suggest to add a class, let's say `.execution-description` and add `color: var(--bs-gray-700);` as a single property, like on this PR [9d1700c#diff-6f86c9142fef96430e757017dc75d0d8674a20b61d1c45a6bbd7478332d84b84R151](https://github.com/kestra-io/kestra/commit/9d1700caf112419acdcbc86d726193874831fe08#diff-6f86c9142fef96430e757017dc75d0d8674a20b61d1c45a6bbd7478332d84b84R151).

I've made the requested changes. Please review them when you get a chance. Thank you!